package.json

```diff
@@ -19,9 +19,6 @@
     "url": "https://github.com/Agoric/Jessie/issues"
   },
   "homepage": "https://github.com/Agoric/Jessie#readme",
-  "dependencies": {
-    "@agoric/harden": "0.0.4"
```

Comment:
`makers.js` still has `import '@agoric/harden'`; is that on purpose?

Reply:
Accident.  I've removed it.